betterproto/plugin.py

```diff
@@ -358,41 +349,25 @@ def generate_code(request, response):
 
         # Fill response
         f = response.file.add()
-        # print(filename, file=sys.stderr)
-        f.name = filename.replace(".", os.path.sep) + ".py"
+        f.name = filename
```

Comment:
in the old implementation, filename contained a sort of package name, like `foo.bar`, and in this line, it was converted to a path.

in the new implementation, filename is already a path, so this transform is not needed.

it made more sense to me decide on the filenames earlier on, not at the very last step, since also the variable is called `filename`